workshop/exercise-6/README.md

```diff
@@ -0,0 +1,86 @@
+## (OPTIONAL, FOR ADVANCED USERS) Set up a Private Container Registry and Obtain Credentials
+
+In the previous exercises, we deployed a container to Knative directly from dockerhub. What if we want to build our own container from source? In this lab, we'll use the [IBM Container Registry](https://console.bluemix.net/docs/services/Registry/registry_overview.html#registry_overview) to host our container images since you already have access to this through your IBM Cloud Account. IBM Container Registry enables you to store and distribute container images in a fully managed private registry.
+
+1. You will need to first create an IBM Container Registry namespace. Let's confirm you're logged in.
+
+    ```
+    ibmcloud login
+    ```
+
+    When prompted, select your own account, and then enter the number for the region `us-south`.
+
+
+1. Add a namespace to your account. You must set at least one namespace to store images in IBM Cloud Container Registry. Choose a unique name for your first namespace. A namespace is a collection of related repositories (which in turn are made up of individual images). You can set up multiple namespaces as well as control access to your namespaces by using IAM policies.
+
+    ```
+    ibmcloud cr namespace-add <my_namespace>
+    ```
+
+2. Create an API key. This API key can be used to automate pushing and pulling of Docker images to and from your namespaces. The automated build processes you'll be setting up will use this key to access your images.
```

Comment:
we should probably use the term "container image" instead of "docker image" throughout since people may not necessarily being using docker to build.

Reply:
agree, updated.